lib/fastlane/actions/pod_push_trunk.rb

```diff
@@ -0,0 +1,54 @@
+module Fastlane
+  module Actions
+    class PodPushTrunkAction < Action
+      def self.run(params)
+        command = 'pod trunk push'
+        if params[:path]
+          command << " '#{params[:path]}'"
+        end
+
+        result = Actions.sh("#{command}")
+        Helper.log.info "Successfully pushed Podspec".green
+        return result
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Push a Podspec to Trunk"
+      end
+
+      def self.details
+        ""
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :path,
+                                       description: "The Podspec you want to push",
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                         raise "Couldn't find file at path '#{value}'".red unless File.exist?(value)
```

Comment:
This block should do two things:

1. Check if the path matches `\.podspec$\`
2. Check if the file exists